test/hotspot/jtreg/runtime/handshake/HandshakeDirectTest.java

```diff
@@ -39,7 +39,7 @@
 
 public class HandshakeDirectTest  implements Runnable {
     static final int WORKING_THREADS = 32;
-    static final int DIRECT_HANDSHAKES_MARK = 50000;
+    static final int DIRECT_HANDSHAKES_MARK = 500000;
```

Comment:
Could this timeout?

Reply:
I have seen no signs of it and have been through several mach5 runs with no issues.